src/cpp/src/visual_language/qwen2vl/classes.cpp

```diff
@@ -112,18 +115,70 @@ std::shared_ptr<ov::Node> create_flatten_patches(std::shared_ptr<ov::Node> input
     return flattened;
 }
 
+std::pair<std::shared_ptr<ov::Model>, std::shared_ptr<ov::op::v0::Result>> patch_preprocess_branch_image(
+    const std::shared_ptr<ov::op::v0::Parameter>& raw_image_1,
+    const std::shared_ptr<ov::op::v0::Parameter>& resize_shape,
+    const std::shared_ptr<ov::op::v0::Constant>& image_mean,
+    const std::shared_ptr<ov::op::v0::Constant>& image_scale,
+    const std::shared_ptr<ov::op::v0::Parameter>& tile_shape) {
+    auto img_f32_nchw = create_f32_nchw_input(raw_image_1);
+    auto img_resized = create_bicubic_resize(img_f32_nchw, resize_shape);
+    auto img_normalized = create_normalization(img_resized, image_mean, image_scale);
+    auto temporal_images = std::make_shared<ov::op::v0::Tile>(img_normalized, tile_shape);
+    auto results = std::make_shared<ov::op::v0::Result>(temporal_images);
+    return {
+        std::make_shared<ov::Model>(results, ov::ParameterVector{raw_image_1, resize_shape, tile_shape}, "then_body"),
+        results};
+}
+
+std::pair<std::shared_ptr<ov::Model>, std::shared_ptr<ov::op::v0::Result>> patch_preprocess_branch_video(
+    std::shared_ptr<ov::op::v0::Parameter> cond_img_vid,
+    std::shared_ptr<ov::op::v0::Parameter> raw_frame_1,
+    std::shared_ptr<ov::op::v0::Parameter> raw_frame_2,
+    std::shared_ptr<ov::op::v0::Parameter> resize_shape,
+    std::shared_ptr<ov::op::v0::Constant> image_mean,
+    std::shared_ptr<ov::op::v0::Constant> image_scale) {
```

Comment:
```suggestion
    const std::shared_ptr<ov::op::v0::Parameter>& cond_img_vid,
    const std::shared_ptr<ov::op::v0::Parameter>& raw_frame_1,
    const std::shared_ptr<ov::op::v0::Parameter>& raw_frame_2,
    const std::shared_ptr<ov::op::v0::Parameter>& resize_shape,
    const std::shared_ptr<ov::op::v0::Constant>& image_mean,
    const std::shared_ptr<ov::op::v0::Constant>& image_scale) {
```
asked in the previous review